new/LIT6910MarkFiveWoman.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6910MarkFiveWoman" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Mark 5,25-29</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
```

Comment:
I think it would be useful if the record contained in the title and abstract a paraphrase of the excerpt's content and possibly some indications on its use (such as "used as protective prayer"), to increase the chances of finding it when one searches with other words than the gospel passage

Reply:
Marrassini wrote that this was a passage from Mark 5 about the healing of haemorrhoids. I do not really understand the passage. All I understand is that it is about a woman whose blood is flowing. I think it refers to vaginal blood, but I am not an expert.
<img width="252" alt="Screenshot 2023_MarkFiveWoman" src="https://github.com/BetaMasaheft/Works/assets/40291787/6d934181-42a2-48b2-906b-fac1f44a2aa9">

Reply:
ask wikipedia https://en.wikipedia.org/wiki/Jesus_healing_the_bleeding_woman, the Bible was not invented by Ethiopians )

Reply:
you can copypaste the abstract from wikipedia